index.js

```diff
@@ -353,5 +364,23 @@ formMobile.addEventListener('submit', (event) => {
     event.preventDefault();
   } else {
     messageMob.textContent = '';
+    const formDataMobile = {
+      name: mobileName.value,
+      email: emailMob.value,
+      text: mobileText.value,
+    };
+    const stringifyFormDataMobile = JSON.stringify(formDataMobile);
+    localStorage.setItem('mobileData', stringifyFormDataMobile);
   }
 });
```

Comment:
- [ ] Ouch! this is so close, but as specified in the project requirements you will have to save the form data in the `local storage`  when the user changes the content of **any input field** and not when the form is submitted so kindly make sure you fix that. ✔
- [ ] This applies to the `formDesktop` event listener as well ✔

Reply:
Issue resolved